src/shogun/preprocessor/RandomFourierGaussPreproc.h

```diff
@@ -16,15 +16,19 @@
 #include <shogun/preprocessor/DensePreprocessor.h>
 
 namespace shogun {
-/** @brief Preprocessor CRandomFourierGaussPreproc
- * implements Random Fourier Features for the Gauss kernel a la Ali Rahimi and Ben Recht Nips2007
- * after preprocessing the features using them in a linear kernel approximates a gaussian kernel
+/** @brief Preprocessor that approximates Gaussian feature map.
  *
- * approximation quality depends on dimension of feature space, NOT on number of data.
+ * Effectively it requires only two parameters for initialization: 
+ * (A) m_kernelwidth: exp(-x^2 / kernel_width)
```

Comment:
no need to state the parameters here, BUT doing a math formula of the kernel and the approximation would be good

Reply:
TexRendr, I call upon thee!

Reply:
check the other classes how to do this, e.g. GaussianKernel